src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,48 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        //Test operation with driver
+        Driver driver = new Driver();
```

Comment:
Better to give a more understandable names for variables. Fix it in all places 
```suggestion
        Driver bob = new Driver();
```